lib/src/browser.dart

```diff
@@ -40,6 +40,10 @@ class Browser {
 
   Version _version;
 
+  void reset() {
```

Comment:
This is mostly for the benefit of testing, since I was running into the issue of wanting to test multiple user agent strings which match internet explorer.  Since we are only referencing a single global instance of `_InternetExplorer` and the version is getting cached on that instance, we would get back wrong results in the tests.
So for example I would parse an IE 9 user agent, then go to parse an IE 8 user agent string and the version would be `9.0.0`. Which is incorrect. So the quick fix was to allow for the version to be uncached.